.github/workflows/test.yml

```
@@ -2,69 +2,12 @@ name: CI
 
 on:
   push:
-    branches: [ main ]
+    branches: [ main, revert-awaken-bit ]
     tags:
```

actionlint (1.7.11)

[error] string should not be empty

(syntax-check)